lib/services/components.js

```diff
@@ -469,8 +473,12 @@ function getStyles(name, slug) {
 function getScripts(name, slug) {
   const site = siteService.sites()[slug],
     assetDir = site && site.assetDir || mediaDir,
-    assetDirLen = assetDir.length,
-    assetPath = site && site.assetPath || '';
+    assetDirLen = assetDir.length;
+  let assetPath = site && site.assetPath || '';
+
+  if (site && site.path && site.assetPath !== site.path) {
+    assetPath = site.path;
+  }
```

Comment:
hmm, so given above, I don't think I was able to get vulture to properly serve its assets using every combination I could on those variables (though I did not mess with assetDir), hence my change.  I feel like somewhere the slug or assetPath are being used incorrectly, requiring my update but could be wrong